grafana/overlays/nerc-ocp-obs/grafanas/grafana-image-renderer-service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: grafana-image-renderer
+  namespace: grafana
+  labels:
+    app: grafana-image-renderer
+spec:
+  ports:
+    - port: 8081
+      targetPort: 8081
```

Comment:
You make your life easier if you use the port name instead of number for the targetPort (because that permits you to change ports in the pod without needing to update the service).

Mostly just an FYI; doesn't need changing.